content/doc/find-help/faq.md

```diff
@@ -220,3 +220,8 @@ This way, even if one data center has an accident, your backup are still safe in
 
 To create add-ons, you need to complete your account information, including your city and ZIP code. 
 For instance, you cannot create a Matomo add-on until you provide these details.
+
+## I get unknown regular requests, is there a problem ? 
+
+For monitoring purposes, our system regularly talks to apps in HTTP to ensure they're operationnal. This ping occurs regularly (every 2mn), and might appear in your logs. This is not a safety risk. 
```

Comment:
`This is not a safety risk. `
Agree with David, maybe deleting this part is okay. We already say that this request come from our monitoring in the sentence before.
Otherwise `this is a legitimate request from our systems` is also good.

Reply:
```suggestion
Our system performs routine health checks by sending HTTP requests to applications every 2 minutes. You may notice these periodic pings in your logs - they're simply part of our standard monitoring process.
```